server/index.js

```diff
@@ -91,6 +91,45 @@ app.post('/api/guests', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.put('/api/users/1', (req, res, next) => {
+  const { userName, partnerName, weddingDate } = req.body;
+  const values = [userName, partnerName, weddingDate];
+  if (!userName || !partnerName || !weddingDate) {
+    res.status(400).json({
+      error: 'userName, partnerName, and weddingDate are all required fields'
+    });
+    return;
+  }
+
+  const sql = `
+  update "users"
+    set "userName" = $1,
+        "partnerName" = $2,
+        "weddingDate" = $3
+      where "userId" = 1
+      returning *
+    `;
+
+  db.query(sql, values)
+    .then(result => {
+      res.status(200).json(result.rows[0]);
+    })
+    .catch(err => next(err));
+});
+
+app.get('/api/users', (req, res, next) => {
+  const sql = `
+  select *
+    from "users"
+    `;
```

Comment:
For now, you can include a `where` clause so the client doesn't have to dig into the array.